mediathread/main/views.py

```diff
@@ -811,7 +814,8 @@ def send_faculty_email(form, faculty_user):
             subject,
             body,
             settings.SERVER_EMAIL,
-            [faculty_user.email])
+            [faculty_user.email],
+            fail_silently=not settings.DEBUG)
```

Comment:
The intention was to not give a 500 error to the user on production. Maybe I should remove this and instead catch the exception.